autorag/nodes/generator/base.py

```diff
@@ -41,6 +41,9 @@ def wrapper(
 					"You can check valid llm names from autorag.generator_models."
 				)
 			batch = kwargs.pop("batch", 16)
+			if llm == "huggingfacellm":
+				kwargs["model_name"] = kwargs.pop("model")
+				kwargs["tokenizer_name"] = kwargs["model_name"]
```

Comment:
I don't know why you set input model_name to tokenizer_name, since there is parameter `tokenizer_name` at LlamaIndex `HuggingfaceLLM`

Reply:
Now understand why. Agree